client/lib/impure-lodash/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import {
```

Comment:
How would it be a test helper? I think the individual modules need to import it instead of lodash. How would you see it be a helper?